integration_tests/20_docker_additional_hosts.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+my_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$my_dir/header.sh" "$my_dir/20_env.toml"
+
+docker build ./plans/additional_hosts/http_server -t http_server
+docker run -it -d -p 8080:8080 --name http_server http_server
+docker network connect testground-control http_server
```

Comment:
I believe we want to call `testground healthcheck --runner local:docker --fix` first,
this will create the control network if it's not there yet.